activerecord/test/cases/relations_test.rb

```diff
@@ -2168,6 +2168,8 @@ def test_where_with_comparison_operator
     assert_equal [9, 10, 11], posts.where("id >=": 9).pluck(:id)
     assert_equal [1, 2], posts.where("id <": 3).pluck(:id)
     assert_equal [1, 2, 3], posts.where("id <=": 3).pluck(:id)
+    assert_not_includes posts.where("id <>": 3).pluck(:id), 3
+    assert_not_includes posts.where("id !=": 3).pluck(:id), 3
```

Comment:
```suggestion
    assert_equal [1, 2], posts.where("id <>": 3).pluck(:id)
    assert_equal [1, 2], posts.where("id !=": 3).pluck(:id)
```

Minor suggestion but I feel like the above is clearer since it ensure that `pluck` is still working correctly. If for some reason `posts.where("id <>": 3)` returns `[]`, the assertion will still pass.